command/testdata/fmt/variable_type_in.tf

```diff
@@ -0,0 +1,27 @@
+variable "a" {
```

Comment:
What do you think about adding an object type variable in here? I ask mostly for completion, as I'm someone who likes to read tests first when looking at code, and it'd be nice (even though the expected behavior seems pretty obvious) to have all that in here as well. You absolutely don't need to block merging on this request if you don't want to.

Reply:
I assume by this you mean some more examples of situations that currently _aren't_ significantly changed by `terraform fmt`, which would then potentially help us as we add additional behaviors in future? If so, that sounds like a great idea and I'll put some in now before I merge this.